src/ventures-for-shared-prosperity/focus-area.njk

```diff
@@ -7,14 +7,114 @@ title: Focus area for ideas42 Venture Studio
 {% block masthead %}
   {{ super() }}
   <div class="hero standard-restricted-width">
-    <div class="g-max-3 g-right">
-      <p class="hero__tag">Where those who wish to focus come</p>
-      <h1 class="hero__title">
-      Focus Area
-      </h1>
+{%include "components/cohort-banner.njk" %}
+        <p class="hero__tag">the poverty tax </p>
+        <h1 class="hero__title">
+          The Excess Costs of Poverty
+        </h1>
+        <p>
+          There are tens of millions of people in the U.S. who face chronic hunger, housing insecurity, inadequate healthcare, and a daily struggle to make ends meet. The causes of poverty are diverse, complex, and often structural, and the consequences are catastrophic at an individual and societal level.
+        </p>
+<div class="g-max-3 g-right">
+      <img src="" alt="Homeless encampment">
+      <img src="" alt="Swirly circle here">
     </div>
+
+      
+
+      
+      <p>
+        Our inaugural cohort of entrepreneurs at 
+        <a href="/ventures-for-shared-prosperity">
+          Ventures for Shared Prosperity
+        </a>
+        , supported by Wells Fargo, will focus on building businesses that improve the lives of millions by reducing the costs for those living in poverty. While most anti-poverty efforts focus (justifiably) on increasing incomes and economic opportunity, an often-ignored part of the equation is the reality that living in poverty in America is 
+        <a href="">
+          incredibly expensive.
+        </a>
+      </p>
+
+      <p>
+        In partnership with our first cohort of entrepreneurs we will work to alleviate the pernicious ways that the “poverty tax” manifests itself. If helping solve this problem is interesting, 
+        
+        <a href="/apply">
+          apply
+        </a>
+        to be an entrepreneur in our first cohort. Read on if you want to learn more about the problem, and where we see opportunities to build large, sustainable, impact businesses.
+      </p>
+
+
+          <h2>The Poverty Tax</h2>
+      <p>In order to identify potential business opportunities, it's important to first understand the many different forms that the poverty tax takes: </p>
+
+
+          <h3>Direct Financial Costs</h3>
+
+        <ul>
+          <li>
+            <strong>Higher Costs:</strong>
```

Comment:
Another tweak on each of these list items. Let's put the `strong` element in each `p` here. I think it makes sense as part of each sentence.

Reply:
Fixed!